microshift.sh

```diff
@@ -53,7 +53,26 @@ ssh-keygen -t ecdsa -b 521 -N "" -f id_ecdsa_crc -C "core"
 # createrepo package is required to create localrepo for microshift and it's dependenices
 # yum-utils package is required for reposync utility to synchronize packages of a remote DNF repository to a local directory
 function configure_host {
-    sudo dnf install -y git osbuild-composer composer-cli ostree rpm-ostree \
```

Comment:
Looks like this is fixed in osbuild-composer 88.3-1.el9_3 https://access.redhat.com/errata/RHBA-2024:0475

Reply:
@cfergeau yes, I removed that commit now.